package.json

```diff
@@ -27,29 +27,67 @@
 	],
 	"scripts": {
 		"postinstall": "npm run bootstrap",
-		"prettier:check": "prettier --check \"packages/*/src/**/*.ts\"",
+		"prettier:check": "wireit",
 		"prettier:write": "prettier --write \"packages/*/src/**/*.ts\"",
 		"postpublish": "lerna run install:safe --scope lit-plugin && echo \"🎉 Published successfully! You can now publish the vscode extension seperately by running 'cd packages/vscode-lit-plugin && npm run publish'\"",
 		"publish": "lerna run prepare && lerna publish --exact && npm run postpublish",
 		"publish:next": "lerna run prepare && echo \"When asked, please choose 'Custom Prerelease' and use default value\" && lerna publish --dist-tag next --preid next --exact && npm run postpublish",
 		"bootstrap": "lerna clean --yes && lerna bootstrap --hoist --ignore lit-plugin && lerna run install:safe --scope lit-plugin && npm run copylink",
-		"lint": "eslint .",
-		"build": "lerna run build --stream",
+		"lint": "wireit",
+		"build": "wireit",
 		"watch": "lerna run watch --parallel --stream",
-		"test": "lerna run test --parallel --stream",
-		"test:all": "lerna run test:all --parallel --stream",
+		"test": "wireit",
 		"test:watch": "lerna run test:watch --parallel --stream",
 		"readme": "lerna run readme && readme generate -i readme.blueprint.md -c readme.config.json",
 		"dev": "cd dev && TSS_DEBUG=5999 code . --disable-extension runem.lit-plugin",
 		"dev:logs": "touch dev/lit-plugin.log && tail -f dev/lit-plugin.log",
-		"copylink": "node scripts.js copylink",
-		"copylink:watch": "nodemon --exec \"npm run copylink\""
+		"copylink": "wireit",
+		"copylink:watch": "nodemon --exec \"npm run copylink\"",
```

Comment:
Could this just now be replaced with `npm run copylink watch`? Then could drop the `nodemon` dependency too.

Reply:
yes, and probably many of the other watch scripts can be removed too, just need to evaluate them more

ava's watch mode is potentially useful though, the ava tests take a little while